### PR TITLE
Prevent user form entering negative numbers into event duration

### LINKED ISSUE #1228: The duration of an eventtype can be negative
### Issue Summary

If you edit your event types the duration can be negative.

### Steps to Reproduce

1. Edit one of your event types
2. Edit the duration time and set a negative number

This will crash you public site and also the specific event page of your public account.

And also zero minutes results in no bookable time slots.

### PR DESCRIPTION
**Pull request by flemming-pr**
Fixes #1228.

I have added basic validation to only add a positive number into the duration field.

### CODE REVIEW

File: pages/event-types/[type].tsx [hunk 1]
```diff
@@ -546,6 +546,7 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
                         }
                         id="length"
                         required
+                        min="1"
```
Comment (emrysal) [inline pages/event-types/[type].tsx:549]: 1 is probably not a great default, a 1 minute meeting is a bit unrealistic. The smallest meeting time could be 5 - 15m?

**Review by emrysal — APPROVED** [review]
Something to think about, but approved

File: pages/event-types/[type].tsx (hunk 1 above)
Comment (emrysal) [reply pages/event-types/[type].tsx:549]:
```suggestion
                        min="5"
```

Comment (flemming-petersen) [reply pages/event-types/[type].tsx:556]: @PeerRich suggested 10 minutes would be good a good min value for the duration.

**Review by PeerRich — APPROVED** [review]

**Pull request merged**